tests/sentry/api/endpoints/test_sentry_app_details.py

```diff
@@ -103,12 +103,16 @@ def test_update_published_app(self):
             data={
                 'name': 'NewName',
                 'webhookUrl': 'https://newurl.com',
+                'redirectUrl': 'https://newredirecturl.com',
+                'isAlertable': True,
             },
             format='json',
         )
         assert response.data['name'] == 'NewName'
         assert response.data['uuid'] == self.published_app.uuid
         assert response.data['webhookUrl'] == 'https://newurl.com'
+        assert response.data['redirectUrl'] == 'https://newredirecturl.com'
+        assert response.data['isAlertable']
```

Comment:
You could probably change this series of `asserts` to:

```python
assert response.data == {
  'name': 'NewName',
  'uuid': self.published_app.uuid,
  'webhookUrl': 'https://newurl.com',
  'redirectIUrl': 'https:newredirecturl.com',
  'isAlertable': True,
}
```

Reply:
I think I didn't do this because of all the extra stuff there is (status, clientId, clientSecret) and we already test those in another test